sample/src/main/java/com/silvercar/sample/MainActivity.kt

```diff
@@ -1,12 +1,77 @@
 package com.silvercar.sample
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import no.finn.unleash.DefaultUnleash
+import no.finn.unleash.Unleash
+import no.finn.unleash.event.UnleashReady
+import no.finn.unleash.event.UnleashSubscriber
+import no.finn.unleash.repository.FeatureToggleResponse
+import no.finn.unleash.repository.ToggleCollection
+import no.finn.unleash.util.UnleashConfig
+import timber.log.Timber
+import java.util.concurrent.TimeUnit
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var unleash: Unleash
+    private lateinit var unleashEnabled: TextView
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        setSupportActionBar(findViewById(R.id.toolbar))
+
+        supportActionBar?.let { setTitle(R.string.app_name) }
+
+        unleashEnabled = findViewById(R.id.unleashEnabled)
+
+        GlobalScope.launch {
+            unleash = initUnleash()
+            val isEnabled = unleash.isEnabled("test-android")
+            unleashEnabled.text = getString(R.string.unleash_test_android_enabled_formatted,
+                isEnabled)
+        }
+    }
+
+    private suspend fun initUnleash(): Unleash {
+        lateinit var unleash: Unleash
+
+        return withContext(Dispatchers.IO) {
+            suspendCoroutine<Unleash> { coroutine ->
+                val config: UnleashConfig = UnleashConfig.builder()
+                    .appName(application.packageName)
+                    .unleashAPI("https://unleash.silvercar.com/api")
```

Comment:
should this be an ENV variable?

Reply:
I could have put it in the `companion object` with `INTERVAL` but it's only used in this one spot right now. I'm waiting to see if I'll need it elsewhere first.